src/main/resources/avro/bdg.avdl

```diff
@@ -748,299 +748,304 @@ record Variant {
-
-   * a = The number of positive strand reads covering the reference allele
-   * b = The number of positive strand reads covering the alternate allele
-   * c = The number of negative strand reads covering the reference allele
-   * d = The number of negative strand reads covering the alternate allele
-
-   This value takes the score:
-   
-   -10 log((a + b)! * (c + d)! * (a + c)! * (b + d)! / (a! b! c! d! n!)
-
-   Where n = a + b + c + d.
-   */
-  union { null, float } fisherStrandBiasPValue = null;
+  // GLE Number=1 Type=String
+  // Genotype likelihoods of heterogeneous ploidy, used in presence of uncertain copy number.
+  // Reserved genotype field key in VCF v4.2, dropped in VCF v4.3
 
+  // GP Number=G Type=Float
   /**
-   The root mean square of the mapping qualities of reads covering this site.
+   Genotype posterior probabilities in the range 0.0 to 1.0 using the same ordering as the
+   genotype likelihood field; one use can be to store imputed genotype probabilities. VCF genotype
+   field reserved key GP, Number=G.
    */
-  union { null, float } rmsMapQ = null;
-  
-  /**
-   The number of reads at this site with mapping quality equal to 0.
-   */
-  union { null, int } mapq0Reads = null;
+  array<float> posteriorProbabilities = [];
 
+  // GQ Number=1 Type=Integer
   /**
-   The Wilcoxon rank-sum test statistic of the mapping quality scores. The mapping
-   quality scores are separated by whether or not the read supported the reference or the
-   alternate allele.
+   Conditional genotype quality, encoded as a phred quality −10log10 p(genotype call is wrong
+   conditioned on the site being variant). VCF genotype field reserved key GQ.
    */
-  union { null, float } mqRankSum = null;
+  union { null, int } quality = null;
 
+  // HQ Number=2 Type=Integer
   /**
-   The Wilcoxon rank-sum test statistic of the position of the base in the read at this site.
-   The positions are separated by whether or not the base supports the reference or the
-   alternate allele.
+   Haplotype quality, encoded as two phred qualities. VCF genotype field reserved key HQ, Number=2.
    */
-  union { null, float } readPositionRankSum = null;
+  array<int> haplotypeQuality = [];
 
+  // MQ Number=1 Type=Integer
   /**
-   The log scale prior probabilities of the various genotype states at this site.
-   The number of elements in this array should be equal to the ploidy at this
-   site, plus 1.
+   Root Mean Square (RMS) of the mapping quality of reads for this genotype. VCF genotype field
+   reserved key MQ.
    */
-  array<float> genotypePriors = [];
+  union { null, int } rmsMappingQuality = null;
 
+  // PL Number=G Type=Integer
   /**
-   The log scaled posterior probabilities of the various genotype states at this site,
-   in this sample. The number of elements in this array should be equal to the ploidy at
-   this site, plus 1.
+   Phred-scaled genotype likelihoods rounded to the closest integer, and otherwise defined precisely
```

Comment:
So if we read in VCF with PL specified, we convert and write out VCF with GL specified?  Reasonable, needs to be documented.